src/bundle/Resources/public/js/scripts/helpers/cookies.helper.js

```diff
@@ -0,0 +1,27 @@
+(function (global, doc, eZ) {
+    const setCookie = (name, value, maxAgeDays = 356, path = '/') => {
+        const maxAge = maxAgeDays * 24 * 60 * 60;
+
+        doc.cookie = `${name}=${value};max-age=${maxAge};path=${path}`;
+    };
+    const getCookie = (name) => {
+        const cookieName = name + '=';
+        const decodedCookie = decodeURIComponent(doc.cookie);
+        const cookiesArray = decodedCookie.split(';');
+
+        for (index in cookiesArray) {
```

Comment:
I think `for ... of` would be better.

Reply:
Maybe even change it all to `.find`
```
const cookieValue = cookiesArray.find((cookie) => {
  const cookieString = cookie.trim();
  
  return cookieString.indexOf(cookieName) === 0;
});

return cookieValue ? cookieValue.split('=')[1] : '';
```
or sth like this